packages/common/src/index.test.ts

```
@@ -0,0 +1,45 @@
+vi.mock("pino", () => {
+  const pinoMock = vi.fn(() => ({ info: vi.fn() }));
+  return {
+    __esModule: true,
+    default: pinoMock
+  };
+});
+
+import pino, { type LoggerOptions } from "pino";
+import { createLogger } from "./index";
+
+export {};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)